src/Pivot.elm

```diff
@@ -545,11 +546,18 @@ appendGoR =
     Modify.appendGoR
 
 
-{-| Append a list to the right.
+{-| Append a list to the front of the left.
 -}
-appendList : List a -> Pivot a
-appendList =
-    Modify.appendList
+appendListL : List a -> Pivot a -> Pivot a
+appendListL =
+    Modify.appendListL
+
+
+{-| Append a list to the back of the right.
```

Comment:
I don't understand this description (what is "back"?), and as much as I do I don't think the implementation matches.
This one really calls for an example anyway.

What happens after applying `appendListR [6,7]` on a pivot that looks like `1 2 *3* 4 5` (center is `*3*`), and why should that be what happens?

Same with `appendListL`.

Reply:
Hmm.. maybe it's how I conceptualize it? Because the Right is reversed in implementation, but conceptually it's like a big list.

```elm
pivot : Pivot Int
pivot = Pivot 3 ( [ 1, 2 ], [ 5 , 4 ])

appendListR [6,7]  pivot == Pivot 3 ( [ 1, 2 ], [ 7, 6, 5,  4 ])
appendListR [6,7]  pivot |> getA == [ 1, 2, 3, 4, 5, 6, 7 ]
```

Reply:
Or is that the other way around???

Reply:
I think it is.

Reply:
```elm
pivot : Pivot Int
pivot = Pivot 3 ( [ 2, 1 ], [ 4, 5  ])

appendListR [6,7]  pivot == Pivot 3 ( [ 2, 1 ], [ 4, 5, 6, 7 ])
appendListR [6,7]  pivot |> getA == [ 1, 2, 3, 4, 5, 6, 7 ]
```

Reply:
Ah you thought the right is the one that's reversed, that explains everything.
It's easy to remember which is reversed by remembering the reasoning: so ops like  `goL` and `goR` are quick (constant time) as they only involve the heads of the lists.

Final example looks good.
And this is what you had in mind for `appendListL`, right?

```elm
pivot : Pivot Int
pivot = Pivot 3 ( [ 2, 1 ], [ 4, 5  ] )

appendListL [6,7]  pivot == Pivot 3 ( [ 2, 1, 7, 6 ], [ 4, 5 ] )
appendListL [6,7]  pivot |> getA == [ 6, 7, 1, 2, 3, 4, 5 ]
```

Reply:
Right. I know why it's reversed, I'm just not having a good day for thinking. Haha. This implementation would also make some sense, but is probably more confusing.

```elm
appendListL [6,7]  pivot == Pivot 3 ( [ 2, 1, 7, 6 ], [ 4, 5 ] )
appendListL [6,7]  pivot |> getA == [ 7, 6, 1, 2, 3, 4, 5 ]
```

Reply:
Yep, I agree. It makes some sense, but most likely less intuitive. Append should work like `++` where one of the two sides happens to be a pivot. I even think this would be a good description to put in the README.

Reply:
https://ellie-app.com/33vhCYWZG6Ba1

Reply:
So I'm not sure on how the new tests work, but here's ^ my implementations.

Reply:
But I could use help on the wording because I'm not great at that.